File: java/src/adapter/ObjectAdapterExample/MyApp.java
package adapter.ObjectAdapterExample;

public class MyApp {
	Vender vender;			// Vender 객체 인스턴스

	public MyApp(Vender vender) {
		this.vender = vender;     // vender 멤버변수 초기화
	}
	public void run() { 
		vender.request();	      // vender의 request() 호출
	}
}

File: src/adapter/ClassAdapterExample/MyApp.kt
package adapter.ClassAdapterExample

class MyApp // vender 멤버변수 초기화
    (// Vender 객체 인스턴스
    var vender: Vender
) {
    fun run() {
        vender.request() // vender의 request() 호출
    }
}
